dbt/scripts/export_models.py

```diff
@@ -107,12 +107,8 @@ def parse_args():
         logger.info("Export completed successfully.")
 
     except Exception as e:
-        # The CloudWatch log handler does not include "ERROR - " at the
-        # beginning of log lines, so we need to preface the message with
-        # "ERROR - " to trigger alarms.
-        logger.error(f"ERROR - {e}")
-
-        # Suppress the error if we're uploading to Cloudwatch so that we can
-        # perform cleanup below, but otherwise, raise the error
-        if args.log_to_cloudwatch_group is None:
-            raise
+        logger.error(
+            str(e) or f"{type(e).__name__} raised with no message",
```

Comment:
The CloudWatch handler won't push a log line if it doesn't include a message, so we need to catch the case where there is no message to make sure all errors get pushed to CloudWatch. In the real world (i.e. outside of my testing) we should never encounter an unhandled exception with no error, but we might as well be extra defensive while we're here.